### ISSUE TITLE
ShineWifi-S configuration instructions

### ISSUE THREAD

**Issue by adamcohenrose** [muppet3000/homeassistant-grott#70]
Hi there - thanks for this project!

I've got a ShineWIFI-S, and have configured it to talk to grott, so am sharing the steps I took...

1. On the [Growatt dashboard](https://server.growatt.com/index), scroll down to My Photovoltaic Devices
2. Click the arrow to go to "All Devices"
3. You should see a Data Logger with a serial number and a Device Type of ShineWIFI-S
4. Click "Datalogger Setting" for that Data Logger
5. Set the grott server:
    - If your grott instance has a local domain name (e.g. homeassistant.local) then type this in the "Set Domain" field
    - Otherwise, if it has a reserved IP address on your network, then enter the IP address in the "Set IP(Caution IP transfer with)" field
6. Enter the password – it will be in the format `growattYYYYMMDD` (e.g. `growatt20231117`)
7. Click "Yes"

Hope this is useful :-)

**Comment by perfrank** [comment]
Same procedure with ShineLan-X

**Comment by papabak** [comment]
Adam, currently my Wifi-S is connected to my 2,4Ghz network. Do I have to make a connection first to the network of the dongle (192.268.10.100) or do I make the changes from the Shinephone app?

I found these screens:
![image](https://github.com/muppet3000/homeassistant-grott/assets/97854063/75679aa6-0142-47da-adfa-0bc2dfcc3ee5)

 and

![image](https://github.com/muppet3000/homeassistant-grott/assets/97854063/910f4c64-f09e-4b4b-b4b5-60a679243803)

Hope I am on the right track...

**Comment by typxxi** [comment]
no, you are not on the right track, cause you need to read more carefully as he had mentioned really everything.

If you read 1. then the keyword is the growatt dashboard which you simply should have clicked on.
then your server.growatt.com login would appear

> 1. On the [Growatt dashboard](https://server.growatt.com/index), scroll down to My Photovoltaic Devices
and then there is the next point many people will struggle and click somewhere else on SETTINGS (inverter)

But he has mentioned clearly the ARROW and where to go:

> 2. Click the arrow to go to "All Devices"

which looks in reality like this 
![image](https://github.com/muppet3000/homeassistant-grott/assets/20662430/a0de9dd5-e746-45c4-9c89-3550af026295)

And when all devices have been clicked then this will appear with the SETTINGS of THE DATALOGGER
> 3. You should see a Data Logger with a serial number and a Device Type of ShineWIFI-S

And from there you can open the data logger settings.

Key point is the update option of the data logger or setting the right time and also reducing the reporting time from 5 to 1 minute but therefore you need to have the right register which I have not right now.

If you wanna set the reporting to every = 1 minute then here is a short description to achieve that using your phone and app.

https://diysolarforum.com/threads/growatt-data-reporting-interval-change-question.51680/post-727980

FINALLY: all growatt sticks are 2,4 ghz only which means you CAN ADD NEW ONES ONLY if you have turned of the 5 ghz signal on your router first. Your phone then will automatically connect via 2,4 ghz and then the stick is able to be configured cause this will take over the 2,4 ghz signal / data from your phone. If your phone would be connected via 5 ghz band it will never work which is the same for most iot things that all rely on 2,4 ghz !

Be carefull with datalogger settings via web UI. Do not play around from the Web Ui with registers and do not write anything to the stick nor the inverter that you are not 100% sure cause you can break a stick which is quite cheap to replace or break the inverter. Have seen already 2 which have been broken by users and where usually growatt is not opening a warranty case but want money for a fix which no one should complain about. The fee has been about 150€ or 175 $ a few years ago for a cheap mic 600 which then was about 350€ brand new. Just a reminder and warning. 

If you need an update this is the way to go, same for changing ip adress, also for time and so on but be carefull with everything called register and I mean that very seriously cause a mic 600 register must not fit for the mic 3000 register and do not mix that up with the similiar looking WiFi Stick settings like register 100 on the inverter is NOT the same as register 100 on the stick.

**Comment by typxxi** [comment]
> Hi there - thanks for this project!
> 
> I've got a ShineWIFI-S, and have configured it to talk to grott, so am sharing the steps I took...
> 
> 1. On the [Growatt dashboard](https://server.growatt.com/index), scroll down to My Photovoltaic Devices
> 2. Click the arrow to go to "All Devices"
> 3. You should see a Data Logger with a serial number and a Device Type of ShineWIFI-S
> 4. Click "Datalogger Setting" for that Data Logger
> 5. Set the grott server:
>    
>    * If your grott instance has a local domain name (e.g. homeassistant.local) then type this in the "Set Domain" field
>    * Otherwise, if it has a reserved IP address on your network, then enter the IP address in the "Set IP(Caution IP transfer with)" field
> 6. Enter the password – it will be in the format `growattYYYYMMDD` (e.g. `growatt20231117`)
> 7. Click "Yes"
> 
> Hope this is useful :-)

Might be time to close this issue as it is no really one and has not been cause it is well described on grott in the rerouting chapter.

**Comment by muppet3000** [comment]
> > Hi there - thanks for this project!
> > I've got a ShineWIFI-S, and have configured it to talk to grott, so am sharing the steps I took...
> > 
> > 1. On the [Growatt dashboard](https://server.growatt.com/index), scroll down to My Photovoltaic Devices
> > 2. Click the arrow to go to "All Devices"
> > 3. You should see a Data Logger with a serial number and a Device Type of ShineWIFI-S
> > 4. Click "Datalogger Setting" for that Data Logger
> > 5. Set the grott server:
> >    
> >    * If your grott instance has a local domain name (e.g. homeassistant.local) then type this in the "Set Domain" field
> >    * Otherwise, if it has a reserved IP address on your network, then enter the IP address in the "Set IP(Caution IP transfer with)" field
> > 6. Enter the password – it will be in the format `growattYYYYMMDD` (e.g. `growatt20231117`)
> > 7. Click "Yes"
> > 
> > Hope this is useful :-)
> 
> Might be time to close this issue as it is no really one and has not been cause it is well described on grott in the rerouting chapter.

I need to add this logic to the documentation, once I've done that I'll close this ticket
